velox/docs/functions/spark/decimal.rst

```diff
@@ -49,6 +49,13 @@ with the calculated decimal type.
 Decimal Functions
 -----------------
 
+.. spark:function:: unaryminus(x: decimal(p, s)) -> r: decimal(p, s)
+
+    Returns negated value of x (r = -x). Corresponds to Spark's operator ``-``.
+
+    ::
+        SELECT unaryminus(cast(-9999999999999999999.9999999999999999999 as DECIMAL(38, 19))); -- 9999999999999999999.9999999999999999999
```

Comment:
I run this example in Spark3.5.1, but failed, what's your Spark version, Looks like the newest Spark document also does not describe this function. https://spark.apache.org/docs/latest/api/sql/index.html
```
spark-sql (default)> SELECT unaryminus(cast(-9999999999999999999.9999999999999999999 as DECIMAL(38, 19)));
[UNRESOLVED_ROUTINE] Cannot resolve function `unaryminus` on search path [`system`.`builtin`, `system`.`session`, `spark_catalog`.`default`].; line 1 pos 7
spark-sql (default)> root@sr249:/mnt/DP_disk1/code/velox# spark-sql --version
24/11/12 10:11:23 WARN Utils: Your hostname, sr249 resolves to a loopback address: 127.0.1.1; using 10.0.0.149 instead (on interface eno1)
24/11/12 10:11:23 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
Welcome to
      ____              __
     / __/__  ___ _____/ /__
    _\ \/ _ \/ _ `/ __/  '_/
   /___/ .__/\_,_/_/ /_/\_\   version 3.5.1
      /_/

Using Scala version 2.12.18, OpenJDK 64-Bit Server VM, 1.8.0_422
Branch HEAD
Compiled by user heartsavior on 2024-02-15T11:24:58Z
Revision fd86f85e181fc2dc0f50a096855acf83a6cc5d9c
Url https://github.com/apache/spark
Type --help for more information.

```

Reply:
Please run 
`SELECT -(cast(-9999999999999999999.9999999999999999999 as DECIMAL(38, 19)));`

Reply:
Could you add this example to the document? Indicates it is `-` in Spark

Reply:
It works, perhaps you could change it to this example, either way is ok for me.

Reply:
https://spark.apache.org/docs/3.5.3/api/sql/index.html#negative

Reply:
Same to me, not sure who decide this name, but it already works for other numeric types. 
If rename it to `negative`, we need to change in both velox and gluten.

Reply:
Yes, maybe in a separate PR, we may change the name.